proca/schema.graphql

```diff
@@ -64,6 +66,8 @@ type PublicCampaign implements Campaign {
 
   org: Org!
 
+  targets: [Target]
```

Comment:
I don't see the use case here. @marcinkoziej do you?

Reply:
Well to lost the campaign targets if you do not have cached it in static file